tests/h/search/index_test.py

```diff
@@ -309,3 +326,12 @@ def __eq__(self, search_response):
 @pytest.fixture
 def search(es_client):
     return elasticsearch1_dsl.Search(using=es_client.conn).fields([])
+
+
+def _quote_selector(quote):
+    return {
+        'type': 'TextQuoteSelector',
+        'exact': quote,
+        'prefix': 'something before ',
+        'suffix': ' something after',
+    }
```

Comment:
What do you think about just making `_quote_selector()` an inner function defined in `test_match_annotation_by_quote()`? Since it's only used in one place. We can move it out in future if it needs to be used in more than one place (at which point a helper method or fixture on the test class might be better). I like to try to avoid a long list of private helper functions at the bottom of a test file 'cos I don't think it ends up being very good for readability

Reply:
Are the matching `type`, `prefix` and `suffix` actually necessary here? I can see why matching query text against the quote is a useful feature to have, but can't imagine that anyone's gonna type `"TextQuoteSelector"` into their search query. (I guess I can see why matching against prefix and suffix is useful too but wonder if those are separate tests.)

Reply:
The prefix and suffix are here just to make the data match the real world scenario. The type is potentially important at indexing time in order for the indexer to know which field to copy into the "quote" field.

Reply:
I've inlined `_quote_selector` into this test case.